.gitignore

```diff
@@ -15,3 +15,5 @@ node_modules
 .DS_Store
 
 coverage
+
+*.sublime-*
```

Comment:
For all of our files in every since repo, we like to have a new line at the end of every line. Our linter usually catches that, but since this isn't a `.js` file, it's not run against it. [Here's a pretty good SO answer](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for why it's useful/common. It's unfortunate that Sublime's default is to not add the last `\n`. But you can add `"ensure_newline_at_eof_on_save": true` to your preferences to make sure that Sublime does add it if it doesn't exist. While you're at it, I would also recommend adding `"trim_trailing_white_space_on_save": true` as well since our linter checks for that as well.